examples/IRrecvDumpV3/IRrecvDumpV3.ino

```diff
@@ -132,6 +153,39 @@ void setup() {
   irrecv.setUnknownThreshold(kMinUnknownSize);
 #endif  // DECODE_HASH
   irrecv.enableIRIn();  // Start the receiver
+
+  // Add web service to MDNS
+  MDNS.addService("http", "tcp", 80);
+  MDNS.addService("ws", "tcp", 80);
+
+  // handle index
+  server.on("/", HTTP_ANY, [](AsyncWebServerRequest *request) {
+      request->send(200, "text/html",
+      "<html><head><script>\n"
+      "'use strict';\n"
+      "var d=document,c=console;\n"
+      "var ws = new WebSocket('ws://'+location.hostname+'/ws', ['arduino']);\n"
+      "ws.onopen = function() {ws.send('Connect ' + new Date());};\n"
+      "ws.onerror = function(ev) {c.log('WebSocket Error ', ev);};\n"
+      "ws.onclose = function(ev) {c.log('ws closed');};\n"
+      "ws.onmessage = function(ev) {\n"
+      "  c.log('Server: ', ev.data);\n"
+      "  var p = d.body.appendChild(d.createElement('pre'));\n"
+      "  p.textContent = ev.data;\n"
+      "  p.scrollIntoView({behavior: 'smooth', block: 'end', inline: 'nearest'});\n"
+      "};\n"
+      "</script></head><body>\n"
+      "</body></html>");
+  });
+  ws.onEvent(onWsEvent);
+  server.addHandler(&ws);
+  server.begin();  // start webserver
+}
+
+// helper function to output to both Serial and websockets
+void display(const String str) {
+  Serial.print(str);
+  ws.textAll(str);
 }
```

Comment:
All output that is relevant is intended to go via this function, outputing on both serial and websockets